libnm-core/nm-utils-private.h

```diff
@@ -105,6 +105,16 @@ gboolean    _nm_sriov_vf_parse_vlans (NMSriovVF *vf, const char *str, GError **e
 
 /* JSON to GValue conversion macros */
 
+static inline void
+_nm_auto_gvalue_unset_and_free (GValue **ptr)
+{
```

Comment:
other similar macros have the `nm_auto_<verb>_<object>` form, i.e. `_nm_auto_unset_and_free_gvalue`

Reply:
fixed.